ios/HeliumWidget/HeliumHotspotsWidget/HeliumHotspotsWidgetView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  HeliumBalanceWidgetEntryView.swift
+//  Helium
+//
+//  Created by Luis F. Perrone on 1/18/22.
+//
+
+import WidgetKit
+import SwiftUI
+import Intents
+
+struct HeliumHotspotsWidgetView : View {
+  var entry: HotspotsWidgetProvider.Entry
+  @Environment(\.widgetFamily) var family
+    
+  @ViewBuilder
+  var body: some View {
+    switch family {
+    case .systemMedium:
+      
+      VStack(alignment: .leading) {
+        Spacer()
+        
+        ForEach(0..<min(entry.isPreview ? 3 : entry.hotspots.count, 3)) { i in
+          let rewardObj = entry.rewards.first { reward in
+            reward.address == entry.hotspots[i].address
+          }
+          
+          HeliumHotspotListItemView(Utils.removeDashAndCapitalize(text: entry.isPreview ? "Digital Mauve Swan" : entry.hotspots[i].name), entry.isPreview ? "Northeast 5th Ave, Miami, US" : "\(entry.hotspots[i].geocode.longStreet), \(entry.hotspots[i].geocode.shortCity), \(entry.hotspots[i].geocode.shortCountry)", entry.isPreview ? 0.3 : rewardObj?.reward.floatBalance ?? 0.0, i != min(entry.isPreview ? 3 : entry.hotspots.count, 3) - 1, entry.isPreview ? true : entry.hotspots[i].status.online == "online")
```

Comment:
Maybe for the preview we should just use "Angry Purple Tiger" as the hotspot name (this is the name of the actual naming library) and can just use "San Francisco, CA, US" as the address. Unless there is a benefit to using a real hotspot, lets not call one out.

Reply:
Agreed. It has been addressed.